src/netbase.cpp

```diff
@@ -838,13 +838,13 @@ bool CNetAddr::IsValid() const
     if (IsIPv4())
     {
         // INADDR_NONE
-        uint32_t ipNone = INADDR_NONE;
-        if (memcmp(ip+12, &ipNone, 4) == 0)
+        uint32_t outIpNone = INADDR_NONE;
+        if (memcmp(ip+12, &outIpNone, 4) == 0)
```

Comment:
```suggestion
        uint32_t ipv4None = INADDR_NONE;
        if (memcmp(ip+12, &ipv4None, 4) == 0)
```
and `ipv6None` above on lines 830-831.